app/assets/images/payment_icons/depay.svg

```diff
@@ -0,0 +1 @@
 1.44 0 0 1 .625-.571c.35-.152.73-.225 1.111-.212.243-.004.485.025.72.085.218.05.43.12.635.212l.137.074.17.095v-.487a.93.93 0 0 0-.37-.815 1.714 1.714 0 0 0-.995-.243c-.362.002-.721.07-1.059.201a4.233 4.233 0 0 0-.656.307v-1.376c.261-.134.539-.233.826-.296a5.778 5.778 0 0 1 1.428-.159c.187-.013.374-.013.561 0 .227.024.45.074.667.148.22.074.426.181.613.318.193.146.349.334.455.55.123.276.181.576.17.878v2.72a.92.92 0 0 1-.286.698 1.98 1.98 0 0 1-.9.424 7.038 7.038 0 0 1-1.555.2Zm.254-2.572a1.651 1.651 0 0 0-.825.159.614.614 0 0 0-.286.56.678.678 0 0 0 .127.403.836.836 0 0 0 .328.275.91.91 0 0 0 .434.106 2.73 2.73 0 0 0 .794-.095.392.392 0 0 0 .275-.392v-.889h-.223c-.2-.076-.41-.126-.624-.148v.021Zm4.645 4.688a3.75 3.75 0 0 1-.603 0 2.922 2.922 0 0 1-.486-.148v-1.45l.306.074c.12.011.24.011.36 0a.995.995 0 0 0 .497-.127.931.931 0 0 0 .37-.444l.064-.19-2.391-5.863h2.042l1.27 3.979 1.143-3.979h1.894l-2.19 5.926a5.684 5.684 0 0 1-.678 1.301 2.19 2.19 0 0 1-.751.699c-.26.143-.55.22-.847.222Z"/><path d="M30.9 17.938c-.2 0-.399-.02-.593-.063a1.739 1.739 0 0 1-.434-.116v-1.28l.233.052c.123.01.247.01.37 0 .182 0 .36-.048.519-.138a.994.994 0 0 0 .391-.476c.03-.061.054-.125.074-.19l-2.37-5.82h1.894l1.312 4.116 1.185-4.116h1.81l-2.117 5.851a5.385 5.385 0 0 1-.666 1.291c-.19.282-.443.515-.74.678a1.64 1.64 0 0 1-.869.211Z"/></g><path d="M4.28 11.72c-.58-.78-.8-1.48-.53-1.94.27-.46 1.27-.66 2.5-.38a9.88 9.88 0 0 1 .46-1.71c-1.67-.32-3-.07-3.52.79-.52.86-.09 2.11 1 3.39l.09-.15Z" fill="url(#pi-depay-linear-gradient-1)"/><path d="m4.28 11.72-.09-.13-.13.17.09.11.13-.15ZM6.25 9.4a9.88 9.88 0 0 1 .46-1.71l-.63-.09a10.87 10.87 0 0 0-.48 1.69c.22.025.436.061.65.11Z" opacity=".3"/><path d="M13.85 11.87c-.388.46-.816.885-1.28 1.27.83.94 1.2 1.82.88 2.37-.32.55-.93.59-1.82.49 0 .08-.05.15-.07.23 1.6.28 2.88 0 3.37-.82.49-.82.1-2.21-1.08-3.54Z" fill="url(#pi-depay-linear-gradient-2)"/><path d="M13.85 11.87c-.388.46-.816.885-1.28 1.27l.29.35c.455-.395.874-.83 1.25-1.3-.05-.1-.13-.19-.26-.32ZM11.67 16c0 .08-.05.15-.07.23l.09-.23h-.02Z" opacity=".3"/><path d="M6.2 14.36c-1.19.25-2.09.13-2.41-.4-.47-.81.53-2.31 2.32-3.68v-.27c-2.44 1.81-3.75 4-3 5.29.5.86 1.83 1.11 3.49.8a11.63 11.63 0 0 1-.4-1.74Z" fill="url(#pi-depay-linear-gradient-3)"/><path d="M14.94 8.38c-.5-.85-1.8-1.1-3.42-.81l.07.18c1-.13 1.67 0 1.94.49.37.63-.17 1.69-1.29 2.79v1c0 .491-.027.982-.08 1.47 2.31-1.8 3.5-3.89 2.78-5.12Z" fill="url(#pi-depay-linear-gradient-4)"/><path d="M5.57 14.46c.108.589.266 1.168.47 1.73l.6-.09a11.63 11.63 0 0 1-.44-1.74c-.22.04-.43.08-.63.1Zm.54-4.18v-.27c-.26.19-.51.39-.74.59v.28c.24-.2.48-.4.74-.6Zm6.13.72v1c0 .491-.027.982-.08 1.47l.09-.07c.062-.464.098-.932.11-1.4v-1h-.12Zm-.57-3.26-.07-.19h-.08l.07.18.08.01Z" opacity=".3"/><path d="M9.14 5.16c-1.73 0-3.13 3.06-3.13 6.84s1.4 6.84 3.13 6.84 3.13-3.06 3.13-6.84-1.42-6.84-3.13-6.84Zm.83 12.53c-1.12 0-2.12-2.53-2.12-5.64s1-5.65 2.12-5.65 2.11 2.53 2.11 5.65c0 3.12-.94 5.64-2.11 5.64Z" fill="url(#pi-depay-linear-gradient-5)"/></g></svg>
```

Comment:
Small nitpick: can we remove the `xmlns:xlink="http://www.w3.org/1999/xlink"` attribute in the `<svg>`?

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" width="38" height="24" role="img" aria-labelledby="pi-depay"><title id="pi-depay">DePay</title><defs><linearGradient gradientTransform="translate(-.15)" gradientUnits="userSpaceOnUse" id="pi-depay-linear-gradient-1" x1="5.6" x2="4.24" y1="6.25" y2="11.1"><stop offset=".09" stop-color="#6fa0d3"/><stop offset=".45" stop-color="#447fc0"/><stop offset=".66" stop-color="#3e75af"/><stop offset="1" stop-color="#275280"/></linearGradient><linearGradient gradientTransform="translate(-.15)" gradientUnits="userSpaceOnUse" id="pi-depay-linear-gradient-2" x1="14.82" x2="13.43" y1="10.99" y2="15.93"><stop offset=".09" stop-color="#6fa0d3"/><stop offset=".45" stop-color="#447fc0"/><stop offset=".66" stop-color="#3e75af"/><stop offset="1" stop-color="#275280"/></linearGradient><linearGradient gradientTransform="translate(-.15)" gradientUnits="userSpaceOnUse" id="pi-depay-linear-gradient-3" x1="17.16" x2="1.44" y1="12.27" y2="13.47"><stop offset=".09" stop-color="#fce876"/><stop offset=".52" stop-color="#ffe63f"/><stop offset=".7" stop-color="#e8cf3a"/><stop offset="1" stop-color="#b9a231"/></linearGradient><linearGradient gradientTransform="translate(-.15)" gradientUnits="userSpaceOnUse" id="pi-depay-linear-gradient-4" x1="16.99" x2="1.28" y1="10.12" y2="11.32"><stop offset="0" stop-color="#fce876"/><stop offset=".16" stop-color="#ffe63f"/><stop offset="1" stop-color="#b9a231"/></linearGradient><linearGradient gradientTransform="translate(-.15)" gradientUnits="userSpaceOnUse" id="pi-depay-linear-gradient-5" x1="10.96" x2="7.54" y1="18.33" y2="5.37"><stop offset=".09" stop-color="#cb296b"/><stop offset=".55" stop-color="#ed3a85"/><stop offset="1" stop-color="#f185b5"/></linearGradient></defs><g fill-rule="evenodd"><path d="M3 0h32a3 3 0 0 1 3 3v18a3 3 0 0 1-3 3H3a3 3 0 0 1-3-3V3a3 3 0 0 1 3-3Z" fill-opacity=".07"/><path d="M3 1h32a2 2 0 0 1 2 2v18a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V3a2 2 0 0 1 2-2Z" fill="#fff"/><g fill="#06364c"><path d="M16.9 15.886v-7.64h3.577a3.344 3.344 0 0 1 2.116.592c.535.465.818 1.156.762 1.863.04.73-.242 1.442-.773 1.947a3.005 3.005 0 0 1-2.116.687h-1.587v2.55H16.9Zm1.936-4.032h1.323c.14.002.28-.02.413-.063a.645.645 0 0 0 .317-.223.836.836 0 0 0 .19-.328c.053-.138.079-.286.075-.434a1.323 1.323 0 0 0-.074-.476 1.271 1.271 0 0 0-.18-.36.772.772 0 0 0-.318-.232 1.312 1.312 0 0 0-.423-.074h-1.323v2.19Z"/><path d="M16.953 15.886V8.299h3.524c.73-.037 1.45.168 2.052.582a2.18 2.18 0 0 1 .73 1.82 2.423 2.423 0 0 1-.74 1.904c-.572.473-1.301.715-2.042.678h-1.694v2.603h-1.83Zm1.83-3.926h1.376c.148.004.296-.022.434-.074a.678.678 0 0 0 .339-.222.89.89 0 0 0 .2-.35c.053-.146.078-.3.075-.455a1.448 1.448 0 0 0-.074-.497 1.12 1.12 0 0 0-.19-.37.804.804 0 0 0-.35-.244 1.133 1.133 0 0 0-.434-.084h-1.375v2.296ZM26 15.885a5.023 5.023 0 0 1-1.058-.095 2.34 2.34 0 0 1-.815-.317 1.545 1.545 0 0 1-.529-.603 2.117 2.117 0 0 1-.19-.953c-.01-.298.06-.593.2-.857.149-.264.38-.472.657-.592a2.507 2.507 0 0 1 1.132-.223c.246-.003.492.025.73.085.224.047.44.121.646.222l.137.064.095.053v-.392a.878.878 0 0 0-.349-.772 1.682 1.682 0 0 0-.963-.233c-.361 0-.72.065-1.058.19-.256.098-.5.222-.73.37v-1.46c.271-.15.564-.256.868-.317a5.872 5.872 0 0 1 1.439-.159 4.03 4.03 0 0 1 .571 0c.227.024.45.074.667.148.23.071.444.182.635.329.201.15.365.346.476.571.124.283.182.59.17.9v2.772a.942.942 0 0 1-.308.73c-.264.22-.581.37-.92.434-.496.088-1 .124-1.503.106Zm.254-2.57a1.514 1.514 0 0 0-.794.158.54.54 0 0 0-.264.508c0 .132.04.262.116.37.082.11.187.2.307.265a.91.91 0 0 0 .413.095c.26.015.52-.01.772-.074a.328.328 0 0 0 .244-.339v-.846h-.138a2.946 2.946 0 0 0-.656-.106v-.032Z"/><path d="M26 15.886a4.996 4.996 0 0 1-1.005-.096 2.433 2.433 0 0 1-.794-.307 1.365 1.365 0 0 1-.508-.582c-.135-.29-.2-.61-.19-.93-.008-.292.062-.58.2-.837a1.44 1.44 0 0 1 .625-.571c.35-.152.73-.225 1.111-.212.243-.004.485.025.72.085.218.05.43.12.635.212l.137.074.17.095v-.487a.93.93 0 0 0-.37-.815 1.714 1.714 0 0 0-.995-.243c-.362.002-.721.07-1.059.201a4.233 4.233 0 0 0-.656.307v-1.376c.261-.134.539-.233.826-.296a5.778 5.778 0 0 1 1.428-.159c.187-.013.374-.013.561 0 .227.024.45.074.667.148.22.074.426.181.613.318.193.146.349.334.455.55.123.276.181.576.17.878v2.72a.92.92 0 0 1-.286.698 1.98 1.98 0 0 1-.9.424 7.038 7.038 0 0 1-1.555.2Zm.254-2.572a1.651 1.651 0 0 0-.825.159.614.614 0 0 0-.286.56.678.678 0 0 0 .127.403.836.836 0 0 0 .328.275.91.91 0 0 0 .434.106 2.73 2.73 0 0 0 .794-.095.392.392 0 0 0 .275-.392v-.889h-.223c-.2-.076-.41-.126-.624-.148v.021Zm4.645 4.688a3.75 3.75 0 0 1-.603 0 2.922 2.922 0 0 1-.486-.148v-1.45l.306.074c.12.011.24.011.36 0a.995.995 0 0 0 .497-.127.931.931 0 0 0 .37-.444l.064-.19-2.391-5.863h2.042l1.27 3.979 1.143-3.979h1.894l-2.19 5.926a5.684 5.684 0 0 1-.678 1.301 2.19 2.19 0 0 1-.751.699c-.26.143-.55.22-.847.222Z"/><path d="M30.9 17.938c-.2 0-.399-.02-.593-.063a1.739 1.739 0 0 1-.434-.116v-1.28l.233.052c.123.01.247.01.37 0 .182 0 .36-.048.519-.138a.994.994 0 0 0 .391-.476c.03-.061.054-.125.074-.19l-2.37-5.82h1.894l1.312 4.116 1.185-4.116h1.81l-2.117 5.851a5.385 5.385 0 0 1-.666 1.291c-.19.282-.443.515-.74.678a1.64 1.64 0 0 1-.869.211Z"/></g><path d="M4.28 11.72c-.58-.78-.8-1.48-.53-1.94.27-.46 1.27-.66 2.5-.38a9.88 9.88 0 0 1 .46-1.71c-1.67-.32-3-.07-3.52.79-.52.86-.09 2.11 1 3.39l.09-.15Z" fill="url(#pi-depay-linear-gradient-1)"/><path d="m4.28 11.72-.09-.13-.13.17.09.11.13-.15ZM6.25 9.4a9.88 9.88 0 0 1 .46-1.71l-.63-.09a10.87 10.87 0 0 0-.48 1.69c.22.025.436.061.65.11Z" opacity=".3"/><path d="M13.85 11.87c-.388.46-.816.885-1.28 1.27.83.94 1.2 1.82.88 2.37-.32.55-.93.59-1.82.49 0 .08-.05.15-.07.23 1.6.28 2.88 0 3.37-.82.49-.82.1-2.21-1.08-3.54Z" fill="url(#pi-depay-linear-gradient-2)"/><path d="M13.85 11.87c-.388.46-.816.885-1.28 1.27l.29.35c.455-.395.874-.83 1.25-1.3-.05-.1-.13-.19-.26-.32ZM11.67 16c0 .08-.05.15-.07.23l.09-.23h-.02Z" opacity=".3"/><path d="M6.2 14.36c-1.19.25-2.09.13-2.41-.4-.47-.81.53-2.31 2.32-3.68v-.27c-2.44 1.81-3.75 4-3 5.29.5.86 1.83 1.11 3.49.8a11.63 11.63 0 0 1-.4-1.74Z" fill="url(#pi-depay-linear-gradient-3)"/><path d="M14.94 8.38c-.5-.85-1.8-1.1-3.42-.81l.07.18c1-.13 1.67 0 1.94.49.37.63-.17 1.69-1.29 2.79v1c0 .491-.027.982-.08 1.47 2.31-1.8 3.5-3.89 2.78-5.12Z" fill="url(#pi-depay-linear-gradient-4)"/><path d="M5.57 14.46c.108.589.266 1.168.47 1.73l.6-.09a11.63 11.63 0 0 1-.44-1.74c-.22.04-.43.08-.63.1Zm.54-4.18v-.27c-.26.19-.51.39-.74.59v.28c.24-.2.48-.4.74-.6Zm6.13.72v1c0 .491-.027.982-.08 1.47l.09-.07c.062-.464.098-.932.11-1.4v-1h-.12Zm-.57-3.26-.07-.19h-.08l.07.18.08.01Z" opacity=".3"/><path d="M9.14 5.16c-1.73 0-3.13 3.06-3.13 6.84s1.4 6.84 3.13 6.84 3.13-3.06 3.13-6.84-1.42-6.84-3.13-6.84Zm.83 12.53c-1.12 0-2.12-2.53-2.12-5.64s1-5.65 2.12-5.65 2.11 2.53 2.11 5.65c0 3.12-.94 5.64-2.11 5.64Z" fill="url(#pi-depay-linear-gradient-5)"/></g></svg>
```